### DISCUSSION TITLE
能在自动监听白名单地址栏中增加悬停显示完整路径吗

### DISCUSSION THREAD

**Discussion by q943613777** [shion-app/shion#26] [Ideas]
### 目录地址比较长的时候，后面的显示不出来，不清楚当时选择到了哪个目录。像添加监听程序那样，鼠标悬停在地址栏的时候显示完整地址

![image](https://github.com/shion-app/shion/assets/53282874/f9c332f3-ec1c-4217-a5ab-9d934d2dc5ff)
![image](https://github.com/shion-app/shion/assets/53282874/8fe900c1-8611-43f5-9ff4-3983411dad7f)

**Comment by hanaTsuk1** [comment]
后续加上
